main.go

```diff
@@ -117,10 +116,6 @@ func buildGraph(nextRefresh int64) (string, bool) {
 				time.Sleep(time.Second * 1)
 				return
 			}
-			return buildGraph(secondsToWait)
-		} else if events == nil {
-			fmt.Println("No new data available!")
-			return "", false
```

Comment:
Why removing this check? Is it done in some other place that I missed? What is the behavior on API 304s?